releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.10 - March, 21 2020
+* Supported to trim key prefixes before set key-values into environment variables.
+* Supported to use Azure Key Vault reference.
```

Comment:
Suggested wording for this one : `Added support to resolve key vault references`

Reply:
Added support **to resolve** instead of **of resolving**

Reply:
`Added support for doing`